Week 4/Unit 08/Assignment 2/lab/src/main/java/fpt/lab/repository/EmployeeRepository.java

```diff
@@ -0,0 +1,76 @@
+package fpt.lab.repository;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import fpt.lab.model.Employee;
+
+@Repository
+public class EmployeeRepository {
+    
+    @Autowired
+    @Qualifier("employeeJdbcTemplate")
+    private JdbcTemplate jdbcTemplate;
```

Comment:
Avoid to use field injection

Reply:
Thank you for the correction. I already change it to constructor injection. You can check it [here](https://github.com/affandyfandy/java-irham/blob/feature/week-4-lecture-8/Week%204/Unit%2008/Assignment%202/lab/src/main/java/fpt/lab/repository/ContactRepository.java).